client/src/lib/TinyMCEActionRegistrar.js

```diff
@@ -0,0 +1,48 @@
+/**
+ * Acts as top level action registrar component for tinymce
+ */
+class TinyMCEActionRegistrar {
```

Comment:
@tractorcow @flamerohr We should decide if we call these things `registrar`, `register` or `registry` - we already have `ReactRouteRegister` and `ReducerRegister`. There's design pattern is called [registry](https://martinfowler.com/eaaCatalog/registry.html), described as:

> A well-known object that other objects can use to find common objects and services.

That seems to apply here?